File: python/easy/search_insert_position.py
# Best solution
# def searchInsert(nums, target):
#     print(len([x for x in nums if x < target]))

def searchInsert(nums, target):
    lis = []
    for x in nums:
        if x < target:
            lis.append(x)
    print(len(lis))

n = [1,3,5,6]
m = 7

searchInsert(n, m)

File: javascript/easy/search_insert_position.js
// 35. Search Insert Position
// https://leetcode.com/problems/search-insert-position/

var searchInsert = function(nums, target) {
    if (nums.includes(target)) {
        return nums.indexOf(target);
    } else if (target > nums[nums.length - 1]) {
        return nums.length;
    } else {
        for (var i = 0; i < nums.length; i++) {
            if (nums[i] > target) {
                return i;
            }
        }
    }
};

// console.log(searchInsert([1,3,5,6], 5))
console.log(searchInsert([1,3,5,6], 2))